docs/test_cases/TestSuite24.md

```diff
@@ -70,6 +70,15 @@ The end user can see the details of a reused BIE node that he does not own, and
 #### Test Assertion #24.1.13
 The end user can express a reusing BIE that reuses a BIE in WIP state and owned by a different user.
 
+#### Test Assertion #24.1.14
+The end user won't remove a higher level reference to a reused BIE when removing a different reused BIE at lower level. 
```

Comment:
The end user can remove reused BIE references at any level even if there's another same reused BIE at a different level.